### ISSUE TITLE
Reimplement in Keras

### ISSUE THREAD

**Issue by mrgloom** [szad670401/end-to-end-for-chinese-plate-recognition#18]
I'm trying to reimplement the model in Keras and have some questions about this model:
https://github.com/szad670401/end-to-end-for-chinese-plate-recognition/blob/master/train.py#L108

Fo each character/number we have `fc2n` output in our case you have `n=7` of them and `num_hidden = 65` is number of unique character/number in dictionary.
So as I understand at the output we have 7x65 output (row x col) and each row have only one 1.0 and other values are 0.0 (one hot encoding).

I'm not sure how to deal with matrix, because in ordinary cases like classification we have output as vector and `softmax+categorical_crossentropy` on top.

And what if we have digits for example ['0','1','2','3','4','5','6','7','8','9'] (`num_hidden = 10`) and characters for example ['A','B','C'] (`num_hidden = 3`) how to concat 3 and 10 vectors in single matrix?
 
Can you eleborate on this?

Also seems this project is very similar:
https://github.com/apache/incubator-mxnet/blob/master/example/captcha/mxnet_captcha.R#L13

**Comment by szad670401** [comment]
In Keras , you can implement like this .

```
label_nums = 7 """you labels nums"""
n_class = 34 """ you class nums"""
x = [Dense(n_class, activation='softmax', name='c%d'%(i+1))(x) for i in range(label_nums)]
"""multi-output"""
model = Model(input=input_tensor, output=x)
```

**Comment by mrgloom** [comment]
In this code snippet you suggest multiple output 'heads' on neural network?

I have already implemented different configuration in Keras:
https://github.com/mrgloom/Char-sequence-recognition
